projects/components/src/textarea/textarea.component.scss

```diff
@@ -2,6 +2,12 @@
 
 :host {
   ::ng-deep {
+    &.disabled {
```

Comment:
Not needed inside ng deep, we're adding this class

Reply:
it's necessary to keep it to apply the styles

Reply:
What am I missing? It looks like it's just a regular style on an element of our own template.

Reply:
My bad, I made a mistake when I tested it. Styles moved.